### PR TITLE
amam/add_besquare_link

### PR DESCRIPTION
**Pull request by mustofa-binary**
Changes:

-  add besquare link

## Type of change

-   [ ] Bug fix
-   [x] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

**Review by kevinw-binary — DISMISSED** [review]

**Review by yashim-deriv — DISMISSED** [review]

File: src/components/localization/localized-link.js [hunk 1]
```diff
@@ -114,9 +122,12 @@ export const LocalizedLink = React.forwardRef(({ to, ...props }, ref) => {
             !is_smarttrader_link &&
             !is_deriv_app_link &&
             !is_affiliate_link &&
+            !is_community_link &&
             !is_affiliate_sign_in_link &&
+            !is_besquare_link &&
             !is_blog_link &&
-            !is_community_link
+            !is_community_link &&
```
Comment (suthesan-deriv) [inline src/components/localization/localized-link.js:129]:
```suggestion
```

Comment (suthesan-deriv) [reply src/components/localization/localized-link.js:129]: looks like it's appearing twice?

**Review by suthesan-deriv — DISMISSED** [review]

File: src/components/layout/nav-careers.js [hunk 2]
```diff
@@ -23,13 +21,40 @@ const CareerRight = styled(Flex)`
...
 
+const CareerButton = styled(LinkButton)`
+    @media ${device.tabletS} {
+        font-size: 12px;
+    }
+    @media ${device.mobileL} {
+        font-size: 9px;
+    }
+    @media ${device.mobileM} {
+        font-size: 9px;
```
Comment (sean-binary) [inline src/components/layout/nav-careers.js:32]: Looks like a repeat here from `mobileL`. Don't think `localizedlink` or `linkbutton` has overriding styles.

File: src/components/layout/nav-careers.js [hunk 3]
```diff
@@ -47,43 +72,53 @@ export const NavCareers = () => {
...
-                                aria-label={localize('Locations')}
+                                aria-label={'Locations'}
                                 partiallyActive={true}
                             >
                                 LOCATIONS
-                            </StyledLink>
+                            </CareerLink>
+                            <CareerLink
+                                activeClassName="active"
+                                to="/"
```
Comment (sean-binary) [inline src/components/layout/nav-careers.js:101]: Is this the same as passing an empty string to `to` whenever there is another prop of `is_example_link`? I've done so in the past.

Comment (mustofa-binary) [reply src/components/layout/nav-careers.js:101]: its not the same, its adding `/` in the end of the url, only for consistency purposes

**Review by sean-binary — APPROVED** [review]

**Pull request merged**